fake-server/index.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express")
+const bodyParser = require('body-parser');
+const fakeUsers = require("./fake-users.json");
+const app = express();
+const port = 3000
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post("/", (req, res) => {
+    const { username, password } = req.body || {};
+    var policy = "";

```

Comment:
This isn't being used, so needs removing. Also, I'd suggest using let instead of var - although there is no immediate bug here, it can get confusing if the variable was defined in another scope for whatever reason. Let will prevent from declaring a new variable with the same name.